tests/integration/test_opensearch.py

```diff
@@ -697,3 +722,73 @@ def test_custom_backend(self, httpserver, monkeypatch):
             call_safe(cluster.shutdown)
 
         httpserver.check()
+
+    def test_custom_backend_with_custom_endpoint(
+        self,
+        httpserver,
+        monkeypatch,
+        opensearch_client,
+        opensearch_wait_for_cluster,
+        opensearch_create_domain,
+    ):
+        monkeypatch.setattr(config, "OPENSEARCH_ENDPOINT_STRATEGY", "domain")
+        monkeypatch.setattr(config, "OPENSEARCH_CUSTOM_BACKEND", httpserver.url_for("/"))
+
+        # create fake elasticsearch cluster
+        httpserver.expect_request("/").respond_with_json(
+            {
+                "name": "om",
+                "cluster_name": "opensearch",
+                "cluster_uuid": "gREewvVZR0mIswR-8-6VRQ",
+                "version": {
+                    "number": "7.10.0",
+                    "build_flavor": "default",
+                    "build_type": "tar",
+                    "build_hash": "51e9d6f22758d0374a0f3f5c6e8f3a7997850f96",
+                    "build_date": "2020-11-09T21:30:33.964949Z",
+                    "build_snapshot": False,
+                    "lucene_version": "8.7.0",
+                    "minimum_wire_compatibility_version": "6.8.0",
+                    "minimum_index_compatibility_version": "6.0.0-beta1",
+                },
+                "tagline": "You Know, for Search",
+            }
```

Comment:
This response (and the health response) are oddly specific and imply that they might get outdated with newer versions.
You can just respond with "Custom Backend System" to indicate that the routing is working properly.
The last assert would then just be:
```
assert "Custom Backend System" in response.text
```

Reply:
Will do (honestly I simply copy pasted the existing test, should I simplify that one as well, or just stick to this one?